entity-web/src/main/java/eu/europeana/entity/web/jsonld/JsonLdSerializer.java

```diff
@@ -23,11 +23,11 @@ public JsonLdSerializer() {
         mapper.setDateFormat(df);
     }
 
-    public String serializeToJson(Object object) throws EntityApiRuntimeException {
+    public String serializeToJson(Object object) throws EuropeanaApiException {
         try {
             return mapper.writer().writeValueAsString(object);
         } catch (JsonProcessingException e) {
-            throw new EntityApiRuntimeException("Unexpected exception occurred when serializing!",e);
+            throw new EuropeanaApiException("Unexpected exception occurred when serializing!", e);
```

Comment:
We should use one of the specific Exceptions extending  EuropeanaI18nApiException